src/wp-includes/i18n/class-wp-i18n-translation-file-mo.php

```diff
@@ -0,0 +1,221 @@
+<?php
+/**
+ * I18N: WP_I18n_Translation_File_MO class.
+ *
+ * @package WordPress
+ * @subpackage I18N
+ * @since 6.5.0
+ */
+
+/**
+ * Class WP_I18n_Translation_File_MO.
+ *
+ * @since 6.5.0
+ */
+class WP_I18n_Translation_File_MO extends WP_I18n_Translation_File {
```

Comment:
```suggestion
class WP_Translation_File_MO extends WP_Translation_File {
```

I think we can simplify the naming of all the classes by removing the `I18n` part. Not only because `I18n` reads bad (`I18N` would be my preference) but also because I can't see what other `WP_*_Translation_File` classes core would need. Thoughts?

Reply:
I think I meant `I18N` but no objections to dropping that prefix 👍